themes/default/content/docs/esc/get-started/store-and-retrieve-secrets.md

```diff
@@ -24,13 +24,17 @@ values:
       ciphertext: ZXN....
 ```
 
-You can store and retrieve values in an environment via the Pulumi Cloud console or via the CLI.
+You can store and retrieve values in an environment via one of the following methods:
+
+- the Pulumi Cloud console YAML editor
+- the Pulumi Cloud console Table editor
+- the ESC CLI
 
 ## Store Environment Values
 
-### Store via the console
+### Store via the YAML editor
 
-To store values in your environment, first click on the name of the environment to open its definition editor. You will be presented with a split pane view. The left side is where you will write the definition of your environment configuration, and the right side will show a preview of your configuration in JSON format.
+To store values in your environment using the YAML editor, first click on the name of the environment to open it. You will be presented with a split pane view. The left side is where you will write the definition of your environment configuration, and the right side will show a preview of your configuration in JSON format.
```

Comment:
I don't think we call it the 'YAML editor' view. It's still called the 'Editor view' - I've referred to it as 'YAML-based Code editor view"